diff --git a/node_modules/@across-protocol/sdk/dist/cjs/gasPriceOracle/adapters/ethereum.js b/node_modules/@across-protocol/sdk/dist/cjs/gasPriceOracle/adapters/ethereum.js
index a3861ca..562b04e 100644
--- a/node_modules/@across-protocol/sdk/dist/cjs/gasPriceOracle/adapters/ethereum.js
+++ b/node_modules/@across-protocol/sdk/dist/cjs/gasPriceOracle/adapters/ethereum.js
@@ -15,7 +15,7 @@ function eip1559Raw(provider, chainId, baseFeeMultiplier, priorityFeeMultiplier)
         return tslib_1.__generator(this, function (_b) {
             switch (_b.label) {
                 case 0: return [4, Promise.all([
-                        provider.getBlock("pending"),
+                        provider.getBlock("latest"),
                         provider.send("eth_maxPriorityFeePerGas", []),
                     ])];
                 case 1:
