diff --git a/node_modules/@across-protocol/sdk/dist/cjs/utils/index.js b/node_modules/@across-protocol/sdk/dist/cjs/utils/index.js
index c68d54f..e748391 100644
--- a/node_modules/@across-protocol/sdk/dist/cjs/utils/index.js
+++ b/node_modules/@across-protocol/sdk/dist/cjs/utils/index.js
@@ -31,5 +31,5 @@ tslib_1.__exportStar(require("./ValidatorUtils"), exports);
 tslib_1.__exportStar(require("./AddressUtils"), exports);
 tslib_1.__exportStar(require("./SpokeUtils"), exports);
 tslib_1.__exportStar(require("./CCTPUtils"), exports);
-tslib_1.__exportStar(require("./HyperLiquidUtils"), exports);
+// tslib_1.__exportStar(require("./HyperLiquidUtils"), exports);
 //# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/node_modules/@across-protocol/sdk/dist/esm/utils/index.js b/node_modules/@across-protocol/sdk/dist/esm/utils/index.js
index 7b82231..024903f 100644
--- a/node_modules/@across-protocol/sdk/dist/esm/utils/index.js
+++ b/node_modules/@across-protocol/sdk/dist/esm/utils/index.js
@@ -28,5 +28,5 @@ export * from "./ValidatorUtils";
 export * from "./AddressUtils";
 export * from "./SpokeUtils";
 export * from "./CCTPUtils";
-export * from "./HyperLiquidUtils";
+// export * from "./HyperLiquidUtils";
 //# sourceMappingURL=index.js.map
\ No newline at end of file
