diff --git a/node_modules/@web3-onboard/walletconnect/dist/v2.js b/node_modules/@web3-onboard/walletconnect/dist/v2.js
index e24edd8..459a4ab 100644
--- a/node_modules/@web3-onboard/walletconnect/dist/v2.js
+++ b/node_modules/@web3-onboard/walletconnect/dist/v2.js
@@ -48,7 +48,7 @@ function walletConnect(options) {
                     ? // @ts-ignore
                         // Required as WC package does not support hex numbers
                         requiredChains.map(chainID => parseInt(chainID))
-                    : [1];
+                    : [];
                 // Defaults to the chains provided within the web3-onboard init chain property
                 const optionalChainsParsed = Array.isArray(optionalChains) &&
                     optionalChains.length &&
